video/out/vo_gpu_next.c

```diff
@@ -1048,7 +1050,7 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         p->last_hdr_metadata = (struct pl_hdr_metadata){0};
     }
 
-    p->is_interpolated = mix.num_frames > 1;
+    p->is_interpolated = can_interpolate && mix.num_frames > 1;
```

Comment:
Can be simplified to `opts->interpolation && mix.num_frames > 1`. Or if you want to be really pedantic, you could also check that `frame->vsync_offset` is nonzero.

Reply:
Sure.